Java/validation/src/main/resources/application.yml

```diff
@@ -52,6 +62,17 @@ spring:
     multipart:
       max-file-size: 10MB
       max-request-size: 100MB
+      
+  data:
+    redis:
+      host: localhost
+      port: 6379
+      password: suT1ChwYtkwWfbV1bUfRvQZO1YNhNRMj
```

Comment:
Likely security scan issue waiting to happen (even though not a legit vulnerability).